tests/async-unit/test_fax_followup_emails.py

```
@@ -180,46 +192,56 @@ def test_fax_actor_passes_missing_destination_to_template(
+            ):
                 found_missing_destination = True
                 break
 
-        assert found_missing_destination, \
-            "FaxActor._update_fax_for_sent must include 'missing_destination' in the email context (not commented out)"
+        assert (
+            found_missing_destination
+        ), "FaxActor._update_fax_for_sent must include 'missing_destination' in the email context (not commented out)"
         print("✓ FaxActor passes missing_destination to email template")
 
     def test_fax_actor_context_dict_structure(self, test_fax_with_destination):
```

Ruff (0.14.10)

Unused method argument: `test_fax_with_destination`

(ARG002)